azure-pipelines.yml

```diff
@@ -0,0 +1,109 @@
+trigger:
+  - master
```

Comment:
If people fork this repo, I think this would only trigger CI runs on their `master` branch; ideally we would default to all branches (which I think would happen if we removed these lines)

Reply:
Good check, this seems to be entirely unnecessary.